magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -31,8 +29,6 @@ public class NavigationDelegate(
   private val navigationPropagator = NavigationPropagator
   public val backStack: Deque<NavigationEvent> = ArrayDeque()
 
-  private val lifecycleLimiter by attachFieldToLifecycle(LifecycleLimiter())
```

Comment:
this is what spurred me to look into this change. Since NavigationDelegate extends LifecycleAwareComponent, it already has a LifecycleRegistry. Creating a lifecycleLimiter on top of that resulted in two `LifecycleRegistry`s for each NavigationDelegate